blog-service/2024-02-07-collection.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Azure Metrics Collection (Collection)
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - collection
+  - azure-metrics
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We're happy to announce an Azure Metrics Function update that is part of the Azure metric collection framework. Here are some of the key features that this update offers:
```

Comment:
since multiple resources were updated including function app, application insights we cannot just call it an azure metrics function update
`We're happy to announce a update of the Azure Monitor Metrics Collection`